doc/manual/performance-tips.rst

```diff
@@ -1114,6 +1114,53 @@ properties:
    LLVM auto-vectorization may kick in automatically, leading to no further
    speedup with :obj:`@simd`.
 
+While :obj:`@simd` needs to be placed in front of a loop, that is not the case
+for :obj:`@inbounds`. It can be placed in front of any expression as long as it
+is inside a function. The only place where it can be outside of a function
+definition is in front of a loop.
```

Comment:
This information (outside functions only in front of loops) is news to me. Does this mean that a stand-along statement
```Julia
@inbounds a[1] = 0
```
is malformed?